documentation/source/install/on_premise_solutions.rst

```diff
@@ -25,7 +25,9 @@ The following procedure helps you setup and use OpenEBS on a local machine:
     openebs@openebs:~$ cd openebs/k8s/lib/vagrant/test/k8s/1.6
     openebs@openebs:~/openebs/k8s/lib/vagrant/test/k8s/1.6$ vagrant up
 
-It will bring up one kubemaster and two kubeminions.
+It will bring up one kubemaster and two nodes.
+
+**Note:** If vagrant up displays the *Stderr: VBoxManage: error: VT-x is not available (VERR_VMX_NO_VMX)* error, ensure that you have enabled your VM box.
```

Comment:
"ensure that you have enabled your VM box." -> I think this needs a change.. get the exact name/option used in BIOS..

Reply:
It will bring up one kubemaster and two nodes = It will bring up a three node kubernetes cluster with one master and two nodes.